Sources/DolbyIORTSUIKit/Private/ViewModels/StreamViewModel.swift

```diff
@@ -287,7 +287,11 @@ final class StreamViewModel: ObservableObject {
             detailSourceAndViewRenderers: existingDetailSourceAndViewRenderers,
             settings: _
         ):
-            selectedVideoSource = currentlySelectedVideoSource
+            if selectedVideoSource = sources.first { $0.id == currentlySelectedVideoSource.id } ?? sortedSources[0] {
+                selectedVideoSource = currentlySelectedVideoSource
+            } else {
+                selectedVideoSource = sortedSources[0]
+            }
```

Comment:
Actually we only need this one line ?

selectedVideoSource = sources.first { $0.id == currentlySelectedVideoSource.id } ?? sortedSources[0]

instead of 

` 
if selectedVideoSource = sources.first { $0.id == currentlySelectedVideoSource.id } ?? sortedSources[0] {
                selectedVideoSource = currentlySelectedVideoSource
            } else {
                selectedVideoSource = sortedSources[0]
            }
`

Reply:
Right, silly me.